public/views/index.html

```diff
@@ -14,6 +14,15 @@
 
     <base href="[[.AppSubUrl]]/" />
 
+    <script>
+      // NI fork: Ensure that grafana is loaded within an iframe unless it is running locally
+      if (window.self === window.top && window.location.hostname !== 'localhost') {
```

Comment:
is this logic inverted? We only want to redirect if `window.self !== window.top` right?